ui/pages/confirmations/components/confirm/info/nft-token-transfer/nft-token-transfer.tsx

```diff
@@ -13,7 +13,8 @@ const NFTTokenTransferInfo = () => {
   const { currentConfirmation: transactionMeta } =
     useConfirmContext<TransactionMeta>();
 
-  const isWalletInitiated = transactionMeta.origin === 'metamask';
+  // const isWalletInitiated = transactionMeta.origin === 'metamask';
```

Comment:
should we uncomment this line and remove line 17?

Reply:
Good catch!